### ISSUE TITLE
Ask before overwriting an existing tour

### ISSUE THREAD

**Issue by Abhilash-Chandran** [vsls-contrib/codetour#33]
I created a code tour with about three steps. Then I stopped recording. And I wanted to add a new step to the tour and I couldn't find a option to it. So I assumed if I type the same name for the new tour it would ask me to edit it. But instead the existing one got deleted and I lost all the recordings. 

Later in the docs I found how to edit the tour. But overwriting existing one without a warning message is not intuitive IMHO. :-)

BTW: I love this extension. I really hope this gets into other editors as well. Thanks.. 👍 💯

**Comment by lostintangent** [comment]
Doh! This is a good catch 👍 I’ll fix this issue this evening. This wasn’t intentional behavior, it just _happens_ to work that way 😁

BTW, if you have feedback on how I could make it more intuitive to edit tours, I’d love to hear that. Thanks!

**Comment by Abhilash-Chandran** [comment]
The right click context menu is good for me. 
I still see two options.
1. Since we start recording using `ctrl+shift+p` option it could have another option say `Edit Recording` . I think this is the approach I often see in many extensions and I think its easy to show the list of recording as list and let the user choose one of them. 
2. If you think the above will be confusing, then an edit icon along side the play icon for each recording would make sense.  This could be immediately intuitive. But either the context menu or the icon button should exist and not both to avoid confusion.

Having too many option will also confuse users. 😄

**Comment by lostintangent** [comment]
Sounds good! I'll add the `CodeTour: Edit Tour` command as part of this fix. Additionally, in case you didn't notice, there's an "edit" button in the comment UI when replaying a tour. It's the little pencil icon next to the stop button.

**Comment by Abhilash-Chandran** [comment]
What I meant by the second option is here as shown in the fig.
![image](https://user-images.githubusercontent.com/4489849/76795005-89214300-67c8-11ea-9e96-a0afe4a51d13.png)

**Comment by lostintangent** [comment]
This bug has been fixed in the latest extension version. Now, when you record a tour, it will detect if there's already tour with the specified title, and ask if you want to re-enter the title or overwrite the existing tour. Additionally, I added the `CodeTour: Edit Tour` command as well. Thanks for all your feedback, and let me know if there's other things I can do to improve the experience further 👍
